public/400.html

```diff
@@ -52,9 +40,9 @@
                 <div id="wrapper" class="group live">
                     <section id="content" role="main">
                         <header class="page-header group">
-                            <h1 class="heading-xlarge">This page cannot be found</h1>
+                            <h1 class="heading-xlarge">Sorry, there is a problem with the service</h1>
                         </header>
-                        <p>Check that you’ve entered the correct web address or go to your Workbasket.</p>
+                        <p>Please contact <a href="mailto:correspondence@digital.justice.gov.uk?subject=400">correspondence@digital.justice.gov.uk</a> with the details on how you got here to report this issue.</p>
```

Comment:
Could having the subject as just "400" be a little ambiguous for a user? Would they know that that meant perhaps "400 error report" or something might be clearer.

Reply:
Good point - I was thinking more from a dev perspective but you're right, I'll add "error report" as that sounds better.

Reply:
Done.